### ISSUE TITLE
Set different environment with @cronitor.job decorator

### ISSUE THREAD

**Issue by jonchenn** [cronitorio/cronitor-python#21]
Hi there, my understanding is that I can send telemetry metrics to the same dashboard with different environments (staging, prod, etc).

I wonder if there's a way to set a different environment using @cronitor.job decorator in python?
E.g.:

```
@cronitor.job('test-monitor', env="test")
def my_job_func():
   ...
```

Thanks!

**Comment by danielnieto** [comment]
the current implementation does not allow for passing an `env` param, but you can use `CRONITOR_ENVIRONMENT` env var and it will pick it up automatically

**Comment by aflanagan** [comment]
You can also set the env directly on the module
```
cronitor.environment = 'staging'
```

however, I also have included this update in the 4.6.0 release as well!

```
@cronitor.job('test-monitor', env="test")
def my_job_func():
    ...
```
